modules/rosetta/src/main/scala/org/tessellation/rosetta/api/dag/network/NetworkChainObjectStatus.scala

```diff
@@ -0,0 +1,5 @@
+package org.tessellation.rosetta.api.dag.network
+
+import org.tessellation.rosetta.api.generated.dag.network.ChainObjectStatus
+
+case class NetworkChainObjectStatus(value: ChainObjectStatus)
```

Comment:
what is the purpose of this case class and how it will be used?

Reply:
I will confer with @ryle-goehausen and follow-up with you.

Reply:
This serves the same purpose as `TransactionOperationStatus` and is redundant. It has been removed.